### PR TITLE
TE-2253 Responsive image defaults to lazyload

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-2253)

### What **one** thing does this PR do?

Makes all `ResponsiveImage` components to have lazyload by default. 

### Featured Property
featured_properties

### Featured RoomType
featured_roomtype

### Hero
hero

### Host Profile
host_profile

### Pictures
pictures

### Responsive Image
responsive_image

### Promotion 
promotion

### Thumbnail
thumbnail

### Homepage Hero
homepage_hero

### PropertyPageHero
propertypage_

### CODE REVIEW

File: src/components/layout/LazyLoader/component.js [hunk 1]
```diff
@@ -88,7 +88,11 @@ export class Component extends PureComponent {
       lazyProps
     );
 
-    return <div ref={this.createComponentRef}>{componentToRender}</div>;
+    return (
+      <div className="responsive-image-wrapper" ref={this.createComponentRef}>
```
Comment (marcdavi-es) [inline src/components/layout/LazyLoader/component.js]: We don't want to add a class related to `ResponsiveImage` to `LazyLoader`. `LazyLoader` is a general purpose component.

**Comment by red-runner** [comment]
@marcdavi-es This now works without adding styles 🎉

File: src/components/layout/LazyLoader/component.js [hunk 2]
```diff
@@ -88,7 +88,13 @@ export class Component extends PureComponent {
       lazyProps
     );
 
-    return <div ref={this.createComponentRef}>{componentToRender}</div>;
+    return (
+      <Fragment>
+        <div ref={this.createComponentRef} />
+        {componentToRender}
+        <div ref={this.createComponentRef} />
```
Comment (Jonnygwi) [inline src/components/layout/LazyLoader/component.js]: I don't think you can set a reference to two components like this. The bottom of this component will be that of the first div.. Not the last. I think we need to make two refs and update the `getShouldComponentRender` method.

**Comment by red-runner** [comment]
> One thing inline. Also is the thumbnail gif up to date as it looks like the image isn't centered as it was before the change.

@Jonnygwi You were right about the thumbnail; with the new implementation doesn't happen. Uploaded new gif.

**Pull request merged**